lib/tokenVersions.js

```diff
@@ -227,7 +227,7 @@ async function _getNextVersionId() {
   const projection = {_id: 0, 'tokenVersion.id': 1};
   const collection = database.collections['tokenization-tokenVersion'];
   const records = await collection.find(
-    {}, projection).sort({id: -1}).toArray();
+    {}, {projection}).sort({id: -1}).toArray();
```

Comment:
I see that we can include `sort` in the options along with projection.  https://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#find

I think this is nicer and more compact.  

@JSAssassin @aljones15 @mandyvenables thoughts about using options whenever possible by convention?

Reply:
@mattcollier I agree, consolidating `sort` into `options` makes the most sense, all of the options would then be defined in the same space instead of several lines apart.

Reply:
Updated it.

Reply:
using sort in options seems better.